src/AppSearch/Jobs/IndexAppSearchModels.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Packages\AppSearch\Jobs;
+
+use Baka\Contracts\Queue\QueueableJobInterface;
+use Baka\Jobs\Job;
+use Exception;
+use Gewaer\Models\Storms;
+use Kanvas\Packages\AppSearch\Contracts\SearchableModelsInterface;
+use Phalcon\Di;
+
+class IndexAppSearchModels extends Job implements QueueableJobInterface
+{
+    protected $entity;
+
+    /**
+     * Constructor.
+     *
+     * @param Storms $storm
+     */
+    public function __construct(SearchableModelsInterface $entity)
+    {
+        $this->entity = $entity;
+
+        if (!Di::getDefault()->has('elasticApp')) {
+            throw new Exception('Please configure the elasticApp Provider');
+        }
+    }
+
+    /**
+     * Handle the index of the document.
+     *
+     * @return void
+     */
+    public function handle()
```

Comment:
@kaioken add return type

Reply:
@dasdo for now since handle can return anything its a mixed return that why we dont specify anything at the moment, with php 8 we will replace with : mixed